tik/lib/include/tik/KernelInterface.h

```diff
@@ -6,9 +6,9 @@ namespace TraceAtlas::tik
     class KernelInterface
     {
     public:
-        KernelInterface(int index, llvm::BasicBlock *block);
+        KernelInterface(int index, int block);
```

Comment:
int32_t again

Reply:
you mean int64_t right? to match the annotaters

Reply:
Right. Exactly why I want you to make it verbose.